test/__snapshots__/test/test.ts.md

```diff
@@ -7083,3 +7083,347 @@ Generated by [AVA](https://avajs.dev).
+    export interface Enum {␊
+      stringEnum: "a" | "b" | "c";␊
+      impliedStringEnum: "a" | "b" | "c";␊
+      booleanEnum: true;␊
+      impliedBooleanEnum: true;␊
+      integerEnum: -1 | 0 | 1;␊
+      impliedIntegerEnum: -1 | 0 | 1;␊
+      numberEnum?: -1.1 | 0 | 1.2;␊
+      namedIntegerEnum?: NamedIntegerEnum;␊
+      impliedNamedIntegerEnum: ImpliedNamedIntegerEnum;␊
+      impliedHeterogeneousEnum?: -20.1 | null | "foo" | false;␊
+      namedIntegerEnumTitle: NamedInteger;␊
+      impliedNamedIntegerEnumTitle: ImpliedNamedInteger;␊
+      oneOfNamedEnum: IntegerOneOfNamedEnum | StringOneOfNamedEnum;␊
+      anyOfNamedEnum: IntegerAnyOfNamedEnum | StringAnyOfNamedEnum;␊
+      allOfNamedEnum: IntegerAllOfNamedEnum & StringAllOfNamedEnum;␊
+    }␊
+    ␊
+    export enum NamedIntegerEnum {␊
+      One = 1,␊
+      Two = 2,␊
+      Three = 3␊
+    }␊
+    export enum ImpliedNamedIntegerEnum {␊
+      Four = 4,␊
+      Five = 5,␊
+      Six = 6␊
+    }␊
+    export enum NamedInteger {␊
+      One = 1,␊
+      Two = 2,␊
+      Three = 3␊
+    }␊
+    export enum ImpliedNamedInteger {␊
+      Four = 4,␊
+      Five = 5,␊
+      Six = 6␊
+    }␊
+    export enum IntegerOneOfNamedEnum {␊
+      One = 1,␊
+      Two = 2,␊
+      Three = 3␊
+    }␊
+    export enum StringOneOfNamedEnum {␊
+      Four = "four",␊
+      Five = "five",␊
+      Six = "six"␊
+    }␊
+    export enum IntegerAnyOfNamedEnum {␊
+      One = 1,␊
+      Two = 2,␊
+      Three = 3␊
+    }␊
+    export enum StringAnyOfNamedEnum {␊
+      Four = "four",␊
+      Five = "five",␊
+      Six = "six"␊
+    }␊
+    export enum IntegerAllOfNamedEnum {␊
+      One = 1,␊
+      Two = 2,␊
+      Three = 3␊
+    }␊
+    export enum StringAllOfNamedEnum {␊
+      Four = "four",␊
+      Five = "five",␊
+      Six = "six"␊
+    }␊
+    `
+
+## options.enumGenType.dts.js
+
+> Snapshot 1
+
+    `/* tslint:disable */␊
+    /**␊
+     * This file was automatically generated by json-schema-to-typescript.␊
+     * DO NOT MODIFY IT BY HAND. Instead, modify the source JSONSchema file,␊
+     * and run json-schema-to-typescript to regenerate this file.␊
+     */␊
+    ␊
+    export interface Enum {␊
+      stringEnum: "a" | "b" | "c";␊
+      impliedStringEnum: "a" | "b" | "c";␊
+      booleanEnum: true;␊
+      impliedBooleanEnum: true;␊
+      integerEnum: -1 | 0 | 1;␊
+      impliedIntegerEnum: -1 | 0 | 1;␊
+      numberEnum?: -1.1 | 0 | 1.2;␊
+      namedIntegerEnum?: NamedIntegerEnum;␊
+      impliedNamedIntegerEnum: ImpliedNamedIntegerEnum;␊
+      impliedHeterogeneousEnum?: -20.1 | null | "foo" | false;␊
+      namedIntegerEnumTitle: NamedInteger;␊
+      impliedNamedIntegerEnumTitle: ImpliedNamedInteger;␊
+      oneOfNamedEnum: IntegerOneOfNamedEnum | StringOneOfNamedEnum;␊
+      anyOfNamedEnum: IntegerAnyOfNamedEnum | StringAnyOfNamedEnum;␊
+      allOfNamedEnum: IntegerAllOfNamedEnum & StringAllOfNamedEnum;␊
+    }␊
+    ␊
+    export const NamedIntegerEnum: {␊
+      One: 1;␊
```

Comment:
The code actually generates these as `One: 1,` – but the prettier settings of this project change this to `One: 1;`.